spark2.4/src/main/scala/KafkaReader.scala

```diff
@@ -157,6 +176,7 @@ object KafkaReader {
 //          spliceUrl,
 //          spliceKafkaServers,
 //          spliceKafkaPartitions,
+//          useFlowMarkers,
```

Comment:
Remove the whole commented-out block.

Reply:
Entered Jira [DB-10873](https://splicemachine.atlassian.net/browse/DB-10873) to uncomment the block and to add an input param to choose between using ParallelLoaderInserter or the Loader + Inserter pair in the preceding lines.  They are alternative implementations for inserting the data in SpliceMachine.